specs/avatar.md

```diff
@@ -0,0 +1,105 @@
+# Avatar
+
+## Overview
+
+An avatar is a graphical represention of a user or object.
+
+### Use Cases
+
+A common use case would be to display an image or text (usually initials) of a user or an object.
+
+### Features
+- Image slot: Able to slot in an avatar image
+- Badge slot: Able to slot in a badge component
+- `size`, a numeric value that determines the size of the avatar
+- `shape`, a circle or square shape can be chosen
+- `color`, a hexadecimal color can be provided to determine the coin background color
+- If no image is provided, the initials of the provided name will be displayed
+
+### Prior Art/Examples
+
+- [Fluent UI (Persona)](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona)
+- [Fluent UI](https://fluentsite.z22.web.core.windows.net/components/avatar/definition)
+- [Lighting Design](https://www.lightningdesignsystem.com/components/avatar/)
+- [Evergreen](https://evergreen.segment.com/components/avatar)
+- [Ant Design](https://ant.design/components/avatar/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/design-system/avatar)
+---
+### API
+
+*Component Name*
+- `fast-avatar`
+
+#### Attributes
+|   Name    | Description                                  | Type                |
+|-----------|----------------------------------------------|---------------------|
+| `name`    | Accepts name string for coin display         | `string`            |
+| `shape`   | Determines the avatar coin shape.            | `circle | square`   |
+| `fill`    | Sets the background fill of the avatar coin. | `string: hex color` |
+| `color`   | Sets the color of the avatar coin text.      | `string: hex color` |
+| `size`    | Sets the size of avatar coin.                | `number`            |
```

Comment:
Seems like image source and alt text may be missing here. Also, I think I saw some of it above, but might be worth some further clarity about which attrs are required or which combinations work.

Reply:
Ah yes. An oversight on my part. I'll revise this.

Reply:
This has been revised.